server/src/mds3.init.js

```diff
@@ -451,9 +451,9 @@ function combineFilterAndTid2value(q, ds) {
 }
 
 export async function validate_cumburden(ds) {
-	if (!ds.cohort.cumburden) return
-	if (!ds.cohort.cumburden.files) `missing ds.cohort.cumburden.files`
-	const inputFiles = ds.cohort.cumburden.files
+	if (!ds.cohort?.cumburden) return
```

Comment:
no need to check. the func is called in validate_termdb where ds.cohort{} is valid

Reply:
thanks for pointing out.